src/apprt/gtk/menu.zig

```diff
@@ -0,0 +1,125 @@
+const std = @import("std");
+
+const apprt = @import("../../apprt.zig");
+const App = @import("App.zig");
+const Window = @import("Window.zig");
+const Surface = @import("Surface.zig");
+const Builder = @import("Builder.zig");
+
+const gtk = @import("gtk");
+const gdk = @import("gdk");
+const gio = @import("gio");
+const gobject = @import("gobject");
+
+const log = std.log.scoped(.gtk_menu);
+
+pub fn Menu(
```

Comment:
Too few (zero!) comments in this entire file. The trivial functions don't need doc comments but almost everything else should.

Reply:
Added some comments.